iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -462,7 +493,7 @@ public void testCreateTableWithNotSupportedTypes() {
           "Unsupported Hive type", () -> {
             shell.executeStatement("CREATE EXTERNAL TABLE not_supported_types " +
                 "(not_supported " + notSupportedType + ") " +
-                "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
+                "STORED BY ICEBERG " +
```

Comment:
Did we leave some test cases with the orginal syntax `STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'` to make sure it's still working?

Reply:
Yes, we have plenty of other test cases (the ones calling the `TestTables#createTable()`) using the storage handler syntax.